python/res/fm/ecl/ecl_config.py

```diff
@@ -223,6 +223,12 @@ def _get_run_env(self, eclrun_env):
             eclrun_env.pop("PATH")
 
         env.update(eclrun_env)
+
+        # Having the LSB_JOBID set when executing MPI runs will
```

Comment:
Another option would be to add `LSB_JOBID: ~` in the configuration and drop environment variables mapping to `None`. Then you get to move it out of the code 🤷 I'm not entirely sure which is better 🤔

Reply:
I thought about it, and now I